fibonacci.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+function fibonacci(int $number)
+{
+	$array = [];
+	$array[0]=0;
+	$array[1]=1;
```

Comment:
це можна написати трохи лаконічніше. Оскільки це новий массив і ти хочеш туди додати два елементи, це можна записати просто як:
```suggestion
	$array = [0, 1];
```